### PR TITLE
OrderlyPrivate authentication & signature, and order update

### PR DESCRIPTION
**Pull request by samanthatuhseng**
OrderlyPrivateClient:
- sendAuthentication
- generate signature for authentication
- mapping order update in process message

### CODE REVIEW

File: src/exchanges/OrderlyPrivateClient.ts [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+};
+
+
+export type OrderlyClientOptions = PrivateClientOptions & {
+    sendThrottleMs?: number;
+    testNet?: boolean;
+};
+
+/**
+ * Implements OKEx V3 WebSocket API as defined in
```
Comment (alanwu4321) [inline src/exchanges/OrderlyPrivateClient.ts]: comments here are outdated

File: src/exchanges/OrderlyPrivateClient.ts [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+        super(wssPath, "orderly" as ExchangeId, apiKey, apiSecret, apiPassword, undefined, watcherMs);
+        this.hasPrivateOrders = true;
+        this._sendMessage = throttle(this.__sendMessage.bind(this), sendThrottleMs);
+    }
+
+    /**
+     *
+     * @param subscriptionId
+     * @param channel
+     * @see https://www.okx.com/docs-v5/en/#websocket-api-private-channel-order-channel
```
Comment (alanwu4321) [inline src/exchanges/OrderlyPrivateClient.ts]: outdated comments

File: src/exchanges/OrderlyClient.ts [hunk 3]
```diff
@@ -0,0 +1,670 @@
...
+import { Trade } from "../Trade";
+import * as zlib from "../ZlibUtils";
+
+const pongBuffer = Buffer.from("pong");
+
+export type OrderlyClientOptions = ClientOptions & {
+    sendThrottleMs?: number;
+};
+
+/**
```
Comment (alanwu4321) [inline src/exchanges/OrderlyClient.ts:25]: ditto

**Comment by alanwu4321** [comment]
also clean up functions that are not used

**Comment by samanthatuhseng** [comment]
Hi Alan, I've updated the comments and cleaned up some functions. Let me know if you have further feedback

**Comment by alanwu4321** [comment]
@samanthatuhseng  yarn lock conflict should be resolved - I would recommend deleting yarn.lock and then run `yarn install` after rebasing master

**Pull request merged**